lib/graphiti/adapters/abstract.rb

```diff
@@ -400,6 +400,13 @@ def destroy(model_instance)
         raise "you must override #destroy in an adapter subclass"
       end
 
+      def apply_includes_on_scope(scope, sideloads)
+      end
+
+      def clear_active_connections!
```

Comment:
in addition to what we discussed rename this method
```suggestion
      def close
```